marray/tests/test_marray.py

```diff
@@ -20,6 +34,22 @@
 dtypes_all = dtypes_boolean + dtypes_integral + dtypes_real + dtypes_complex
 
 
+def as_numpy(x):
+    # Use `cupy.testing` when `assert_allclose` and `assert_equal` support `strict`
+    try:
+        return np.asarray(x)
+    except:
+        return cupy.asnumpy(x)
+
+
+def pass_backend(*, xp, pass_xp, fun=None, pass_funs=None, dtype=None,
+                 pass_dtypes=None, pass_using=pytest.skip, reason="Debug later."):
```

Comment:
Useful for skipping or xfailing a test for a particular function, dtype, and backend. There are a handful of cases where this sort of thing is needed, such as when there are arithmetic differences between backends that have nothing to do with `MArray`.